### PR TITLE
Time optimization

### PR DESCRIPTION
**Pull request by Iva-khar**
Ratu - using bulk for street table, using dict. for region table;
Ruo - using bulk for founders table;
Using dict. for kved and state, kved and state have one table for Ruo and Rfop;
Rfop - using bulk for rfop table;
Added BulkCreateManager class

### CODE REVIEW

File: ratu/services/main.py [hunk 1]
```diff
@@ -71,6 +73,46 @@ def process(self):
...
                 root.clear()
+        self.bulk_mgr.done()
         print('All the records have been rewritten.')
 
-    print('Converter has imported.')
\ No newline at end of file
+    print('Converter has imported.')
+
+class BulkCreateManager(object):
+    """
```
Comment (criticus) [inline ratu/services/main.py:82]: please add a link to https://www.caktusgroup.com/blog/2019/01/09/django-bulk-inserts/ so we know where this file is from

File: ratu/services/ratu_service.py [hunk 2]
```diff
@@ -29,10 +29,12 @@ class RatuConverter(Converter):
     }
  
     #creating lists for registration items that had writed to db 
-    region_list = []
+    region_dict = {}
     district_list = list()
     city_list = list()
     citydistrict_list = list()
+
+    bulk_mgr = BulkCreateManager(chunk_size=200)
```
Comment (criticus) [inline ratu/services/ratu_service.py:37]: please rename it to `bulk_manager`

File: ratu/services/ratu_service.py [hunk 3]
```diff
@@ -29,10 +29,12 @@ class RatuConverter(Converter):
     }
  
     #creating lists for registration items that had writed to db 
-    region_list = []
+    region_dict = {}
```
Comment (criticus) [inline ratu/services/ratu_service.py:32]: please add a comment here to explain why we use dict for some variables and list for others

File: ratu/services/rfop_service.py [hunk 4]
```diff
@@ -1,20 +1,18 @@
...
-from ratu.services.main import Converter
+from ratu.models.rfop_models import Rfop
+from ratu.models.ruo_models import Kved, State
+from ratu.services.main import Converter, BulkCreateManager
 
 class RfopConverter(Converter):
     
     #paths for remote and local source files
-    FILE_URL = config.FILE_URL
+    FILE_URL = config.FILE_URL_RUO
```
Comment (criticus) [inline ratu/services/rfop_service.py:9]: why inside rfop_service the file_url is called file_url_ruo?

File: ratu/services/rfop_service.py [hunk 5]
```diff
@@ -27,60 +25,65 @@ class RfopConverter(Converter):
     }
     
     #creating list for registration items that had writed to db
-    state_list=[]
-    kved_list=[]
+    state_dict={}
+    kved_dict={}
+
+    bulk_mgr = BulkCreateManager(chunk_size=200)
```
Comment (criticus) [inline ratu/services/rfop_service.py:31]: please rename to `bulk_manager`

File: ratu/services/rfop_service.py [hunk 6]
```diff
@@ -27,60 +25,65 @@ class RfopConverter(Converter):
     }
     
     #creating list for registration items that had writed to db
```
Comment (criticus) [inline ratu/services/rfop_service.py:27]: comment says `creating list` but you're creating dictionaries

File: ratu/services/ruo_service.py [hunk 7]
```diff
@@ -32,56 +31,58 @@ class RuoConverter(Converter):
     }
 
     #creating list for registration items that had writed to db
```
Comment (criticus) [inline ratu/services/ruo_service.py:33]: same here lists and dict

File: ratu/services/main.py (hunk 1 above)
Comment (Iva-khar) [reply ratu/services/main.py:82]: done

File: ratu/services/ratu_service.py (hunk 2 above)
Comment (Iva-khar) [reply ratu/services/ratu_service.py:37]: done

File: ratu/services/rfop_service.py (hunk 4 above)
Comment (Iva-khar) [reply ratu/services/rfop_service.py:9]: It's same source .zip file for Rfop & Ruo

File: ratu/services/rfop_service.py (hunk 5 above)
Comment (Iva-khar) [reply ratu/services/rfop_service.py:31]: done

File: ratu/services/rfop_service.py (hunk 6 above)
Comment (Iva-khar) [reply ratu/services/rfop_service.py:27]: done

File: ratu/services/ruo_service.py (hunk 7 above)
Comment (Iva-khar) [reply ratu/services/ruo_service.py:33]: done

File: ratu/services/ratu_service.py (hunk 3 above)
Comment (Iva-khar) [reply ratu/services/ratu_service.py:32]: done

File: ratu/services/rfop_service.py [hunk 8]
```diff
@@ -26,61 +25,66 @@ class RfopConverter(Converter):
...
     }
     
-    #creating list for registration items that had writed to db
-    state_list=[]
-    kved_list=[]
+    #creating dictionaries for registration items that had writed to db
+    state_dict={} # dictionary uses for keeping whole model class objects
+    kved_dict={}
+
+    bulk_manager = BulkCreateManager(CHUNK_SIZE)
```
Comment (criticus) [inline ratu/services/rfop_service.py:32]: should it be self.CHUNK_SIZE?

Comment (Iva-khar) [reply ratu/services/rfop_service.py:32]: No. This constant is in same class with their initial

**Pull request merged**